### ISSUE TITLE
Tabs are not clickable by keyboard

### ISSUE THREAD

**Issue by Kyzyl-ool** [diplodoc-platform/components#169]
When tab is focused via keyboard, one cannot click it via Enter/Spacebar

**Comment by martyanovandrey** [comment]
I can't reproduce the problem, when the tab is in focus, it already opens automatically
